environment.integration

```diff
@@ -15,6 +15,9 @@ DSS_GCP_SERVICE_ACCOUNT_NAME="org-humancellatlas-integration"
 DSS_ZONE_NAME="integration.data.humancellatlas.org."
 ACM_CERTIFICATE_IDENTIFIER="390e82ea-a684-49f4-a4b3-857f22cee874"
 DSS_CHECKOUT_BUCKET_OBJECT_VIEWERS="serviceAccount:1037839730885-compute@developer.gserviceaccount.com,serviceAccount:caas-account@broad-dsde-mint-test.iam.gserviceaccount.com,serviceAccount:caas-prod-account-for-test@broad-dsde-mint-test.iam.gserviceaccount.com"
+TF_VAR_PROJECT=dcp
+TF_VAR_SERVICE=dss
+TF_VAR_ENV=${DSS_DEPLOYMENT_STAGE}
```

Comment:
variable names need to be updated.

Environment variables that are the same across all deployments do not need to be included in `environment.*`

Reply:
removed references within these files